rpa-home/modules/ROOT/pages/glossary.adoc

```diff
@@ -120,14 +148,22 @@ A project manager is responsible for the automation of a process across all auto
 
 == R
 
+=== Regular Expression 
+
+A sequence of characters and symbols that specify a match pattern to be searched in text.  
+
+=== Run Configuration 
+
+A schedule that specifies the times and intervals for a process to run.
+
```

Comment:
A Run configuration contains more than just the schedule, it also have a name, set the activity parameter starting values, assign user tasks and assign bots. See https://docs.mulesoft.com/rpa-manager/processautomation-deploy#production-configuration